ui.apps/src/main/content/jcr_root/apps/core/cif/components/structure/navigation/v1/navigation/README.md

```diff
@@ -0,0 +1,72 @@
+<!--
+Copyright 2019 Adobe Systems Incorporated
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Commerce Navigation (v1)
+
+Commerce Navigation component is a server-side component written in HTL, rendering a combined AEM page and commerce catalog navigation tree.
+The category tree details are retrieved from Magento via GraphQL. The main usage of this component would be as part of all page templates which include navigation.
+
+## Features
+
+- Combines AEM page and commerce categories
+- Supports multi-level navigation
+- Optionally can use catalog page as the main anchor for commerce categories
+
+### Use Object
+
+The Navigation component uses the `com.adobe.cq.commerce.core.components.models.navigation.NavigationModel` Sling model as its Use-object.
+
+### Properties
```

Comment:
Some clarification:

This component relies on the properties of the AEM Core WCM Navigation component and these properties should be specified on the component node (see the navigation component in the template definitions):
- navigationRoot
- structureDepth - min 1 , max 10
- skipNavigationRoot - should be "true" or it can be omitted because it defaults to "true"
- collectAllPages - should be "false"

The property "magentoRootCategoryId" and "showMainCategories" is specified on the catalogpage node not on the navigation component node.

Reply:
@LSantha can you add this to the readme directly, please. Thanks.

Reply:
@mhaack , done.